language/Portuguese_Brazil/strings.po

```diff
@@ -94,206 +94,208 @@ msgstr "Desativado"
 
 #: plugins/cheat/init.lua:776
 msgid " added"
-msgstr ""
+msgstr " adicionado"
 
 #: plugins/cheatfind/init.lua:344
 msgid "Save Cheat"
-msgstr ""
+msgstr "Salvar Truque"
 
 #: plugins/cheatfind/init.lua:347
 msgid "Default"
-msgstr ""
+msgstr "Padrão"
 
 #: plugins/cheatfind/init.lua:347
 msgid "Custom"
-msgstr ""
+msgstr "Personalizado"
 
 #: plugins/cheatfind/init.lua:348
 msgid "Cheat Name"
-msgstr ""
+msgstr "Nome do Truque"
 
 #: plugins/cheatfind/init.lua:354 plugins/cheatfind/init.lua:718
 msgid "Default name is "
-msgstr ""
+msgstr "Nome padrão é "
 
 #: plugins/cheatfind/init.lua:362
 msgid "Player"
-msgstr ""
+msgstr "Jogador"
 
 #: plugins/cheatfind/init.lua:367
 msgid "Type"
-msgstr ""
+msgstr "Tipo"
 
 #: plugins/cheatfind/init.lua:373
 msgid "Save"
-msgstr ""
+msgstr "Salvo"
 
 #: plugins/cheatfind/init.lua:399
 msgid "Cheat written to "
-msgstr ""
+msgstr "Truque escrito para "
 
 #: plugins/cheatfind/init.lua:399
 msgid " and added to cheat.simple"
-msgstr ""
+msgstr " e adicionado para truque.simples"
 
 #: plugins/cheatfind/init.lua:407
 msgid "Cheat added to cheat.simple"
-msgstr ""
+msgstr "Truque adicionado para truque.simples"
 
 #: plugins/cheatfind/init.lua:412
 msgid ""
 "Unable to write file\n"
 "Check cheatpath dir exists"
 msgstr ""
+"Incapaz de gravar arquivo\n"
+"Verifique se o caminho do diretório de truques existe"
```

Comment:
"cheatpath" is the name of an options that appears on the command line and in ini files, and hence shouldn't be translated